python/dgl/graphbolt/internal/utils.py

```diff
@@ -12,6 +12,18 @@
 from numpy.lib.format import read_array_header_1_0, read_array_header_2_0
 
 
+def numpy_save_aligned(*args, **kwargs):
+    """A wrapper for numpy.save(), ensures the array is stored 4KiB aligned."""
+    # https://github.com/numpy/numpy/blob/2093a6d5b933f812d15a3de0eafeeb23c61f948a/numpy/lib/format.py#L179
+    has_array_align = hasattr(np.lib.format, "ARRAY_ALIGN")
+    if has_array_align:
+        default_alignment = np.lib.format.ARRAY_ALIGN
+        np.lib.format.ARRAY_ALIGN = 4096
```

Comment:
do we need to expose this value to be configured by caller of this function?

Reply:
Max value is 4K and we need 4K so I don't think it would be useful at all. The worst case is that each file can be 4K larger but we don't have many files.